sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -206,6 +216,19 @@ class RunInference(beam.PTransform[beam.PCollection[ExampleT],
   Args:
       model_handler: An implementation of ModelHandler.
       clock: A clock implementing get_current_time_in_microseconds.
+
+  A transform that takes a PCollection of examples (or features) to be used on
+  an ML model. It will then output inferences (or predictions) for those
+  examples in a PCollection of PredictionResults, containing the input examples
+  and output inferences.
+
+  If examples are paired with keys, it will output a tuple
+  (key, PredictionResult) for each (key, example) input.
+
+  Models for supported frameworks can be loaded via a URI. Supported services
```

Comment:
TODO in  a separate PR:
- proof-read these API bits by TW.
- Supported services can also be used - what is this about?
-   TODO(BEAM-14046): Add and link to help documentation : Let's point to the examples directory.

Reply:
Yep. thanks